LICENSE-3rdparty.csv

```diff
@@ -1,6 +1,5 @@
 Component,Origin,License,Copyright
 lib/datadog/core/vendor/multipart-post,https://github.com/socketry/multipart-post,MIT,"Copyright (c) 2007-2013 Nick Sieger."
-lib/datadog/tracing/contrib/active_record/vendor,https://github.com/rails/rails/,MIT,"Copyright (c) 2005-2018 David Heinemeier Hansson"
```

Comment:
was this an automated process, or did you know you had to manually remove this license?

Reply:
I just kind of remembered it, I think it was luck.
Not sure if it's worth creating a GH Action for such detection.